### DISCUSSION TITLE
using .env per docs not working for me

### DISCUSSION THREAD

**Discussion by dkebler** [docker/buildx#1338] [Q&A] [answered]
Trying to use a .env file to set some environment variables in a compose file but wasn't working so I tried the example and even that does not work for me.   The .env is either ignored or not loaded, why?

https://github.com/docker/buildx/blob/master/docs/guides/bake/compose-file.md#env-file

this is the buildx version I'm using `docker buildx version`
`github.com/docker/buildx v0.8.2-docker 6224def4dd2c3d347eee19db595348c50d7cb491`

**Comment by crazy-max** [comment, marked as answer]
This feature is available since [buildx v0.9.0](https://github.com/docker/buildx/releases/tag/v0.9.0).

**Comment by dkebler** [comment]
I was running almost  the latest docker-ce 20.10.17,   but the latest release 20.10.18 did get the new buildx and now the .env file works
`github.com/docker/buildx v0.9.1-docker ed00243a0ce2a0aee75311b06e32d33b44729689`

https://docs.docker.com/engine/release-notes/
